### DISCUSSION TITLE
Can't download Waydroid

### DISCUSSION THREAD

**Discussion by shrisamar000** [waydroid/waydroid#1430] [Q&A]
hey i can't download waydroid because download speeds are like 30 kbps so please suggest some ideas.

**Comment by HafizBeyy** [comment]
> hey i can't download waydroid because download speeds are like 30 kbps so please suggest some ideas.

Manually install itfrom here https://sourceforge.net/projects/waydroid/ . install the vendor "HALIUM" and system "VANILLA" or "GAPPS" . Then run `sudo waydroid init`

**Reply by Runa-Chin** [reply]
I think it helps, but I also thinks they're need more times to download manually. And now we just need vendor MAINLINE and VANILLA.

**Comment by Runa-Chin** [comment]
> hey i can't download waydroid because download speeds are like 30 kbps so please suggest some ideas.

Hello buddy! I hear you didn't have a good Internet condition. Well, you can find a public speed WiFi and do a installation while you'd connected.
